.github/workflows/deploy-staging.yml

```diff
@@ -17,6 +17,8 @@ jobs:
   build_and_deploy:
     env:
       NUXT_UI_PRO_LICENSE: ${{ secrets.NUXT_UI_PRO_LICENSE }}
+      ALGOLIA_SEARCH_API_KEY: ${{ secrets.ALGOLIA_SEARCH_API_KEY }}
+      ALGOLIA_APPLICATION_ID: ${{ secrets.ALGOLIA_APPLICATION_ID }}
```

Comment:
Just the same comment made here: https://github.com/zksync-sdk/sdk-docs/pull/95/files#r1744393386